modules/toybox/0.8.12/overlay/BUILD

```diff
@@ -0,0 +1,103 @@
+load("@rules_license//rules:license.bzl", "license")
+
+package(
+    default_applicable_licenses = [":license"],
+    default_visibility = ["//visibility:public"],
+)
+
+license(
+    name = "license",
+    package_name = "toybox",
+    license_kinds = ["@rules_license//licenses/spdx:0BSD"],
+)
+
+COPTS = [
+    "-I.",
+    "-funsigned-char",
+    "-w",  # const qualifiers are dropped all over the place
+    "-Wundef",
+    "-Wno-char-subscripts",
+    "-Os",
+    "-ffunction-sections",
+    "-fdata-sections",
+    "-fno-asynchronous-unwind-tables",
+    "-fvisibility=hidden",
+]
+
+cc_library(
+    name = "toybox_helper_lib",
+    srcs = glob([
+        "lib/*.c",
+        "lib/*.h",
+    ]) + [
+        "generated/config.h",
+        "generated/flags.h",
+        "generated/globals.h",
+        "generated/help.h",
+        "generated/newtoys.h",
+        "generated/tags.h",
+        "generated/zhelp.h",
+        "toys.h",
+    ],
+    copts = COPTS,
+    features = [
+        "-parse_headers",
+    ],
+    #includes = INCLUDES,
+    textual_hdrs = [
+        "lib/lib.h",
+        "lib/portability.h",
+    ],
+)
+
+cc_library(
+    name = "toybox_toys_lib",
+    srcs = [
+        "generated/tags.h",
+    ] + glob(
+        [
+            "toys/lsb/*.c",
+            "toys/net/*.c",
+            "toys/other/*.c",
+            "toys/posix/*.c",
+        ],
+        exclude = [
+            "toys/lsb/passwd.c",
+            "toys/other/chcon.c",
+        ],
+    ),
+    copts = COPTS,
+    linkopts = ["-lresolv"],
+    textual_hdrs = [
+        "toys.h",
+    ],
```

Comment:
Do we need them here again? We already have a dependency on `toybox_helper_lib`, which already has them...